### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by nickavv**

### CODE REVIEW

**Comment by ghost** [comment]
## Black Duck Security Report

Merging #2 into master will not change security risk.

### Added Components

Clean:       35

### Removed Components

Clean:       25

[Click here to see full report](https://copilot-valid.blackducksoftware.com/github/repos/nickavv/crowdsocks/pull-requests/2)

**Comment by nickavv** [comment]
comment

File: package.json [hunk 1]
```diff
@@ -19,7 +19,6 @@
   },
   "dependencies": {
     "express": "^4.13.4",
-    "http": "0.0.0",
```
Comment (romeara) [inline package.json:22]: COMMENT

**Comment by ghost** [comment]
## Black Duck Security Report

Merging #2 into master will not change security risk.

### Added Components

Clean:       24

### Removed Components

Clean:       1

[Click here to see full report](https://copilot-valid.blackducksoftware.com/github/repos/nickavv/crowdsocks/pull-requests/2)

**Pull request closed without merging**